src/con_duct/filesize.py

```diff
@@ -0,0 +1,88 @@
+"""
```

Comment:
Let's bring this into `__main__.py` .  

- https://github.com/con/duct/issues/187

Reply:
+1, I'll do this before merge.

Reply:
Done in latest push.